crates/common/tedge_utils/src/inotify.rs

```diff
@@ -0,0 +1,17 @@
+pub use inotify_crate::EventMask;
+pub use inotify_crate::WatchMask;
+use inotify_crate::{EventStream, Inotify};
+use std::path::Path;
+
+pub fn inofity_file_watch_stream(
+    config_file: &Path,
```

Comment:
`path` would be a better name: this can be a directory or a file.

I would also fix the function name.